src/main/kotlin/com/mparticle/kits/UrbanAirshipKit.kt

```diff
@@ -206,7 +202,7 @@ class UrbanAirshipKit : KitIntegration(), KitIntegration.PushListener, KitIntegr
                 .apply()
         }
         if (identityType == configuration?.userIdField) {
-            UAirship.shared().namedUser.id = null
+            UAirship.shared().contact.reset() // Previously setting namedUser to null but now is immutable
```

Comment:
This is basically the same thing as setting it null if the user is only managed through the SDK, but its possible that if the device to a user server side this would reset that now. I think it would be safer to only reset it if its not set through the SDK:
```
if (UAirship.shared().contact.namedUserId != null) {
    UAirship.shared().contact.reset()
}
```